client/blocks/reader-author-link/docs/example.jsx

```diff
@@ -9,16 +9,17 @@ import React from 'react';
 import ReaderAuthorLink from 'blocks/reader-author-link';
 import Card from 'components/card';
 
-export default React.createClass( {
+const ReaderAuthorLinkExample = () => {
 
-	displayName: 'ReaderAuthorLink',
+	const author = { URL: 'http://wpcalypso.wordpress.com', name: 'Barnaby Blogwit' };
 
-	render() {
-		const author = { URL: 'http://wpcalypso.wordpress.com' };
-		return (
-			<Card>
-				<ReaderAuthorLink author={ author }>Author site</ReaderAuthorLink>
-			</Card>
-		);
-	}
-} );
+	return (
+		<Card>
+			<ReaderAuthorLink author={ author }>{ author.name }</ReaderAuthorLink>
```

Comment:
It might be me only, but when I first saw this, it bent my mind a bit to think whether `name` is a necessary property for the component or just for displaying text here. Thus I personally feel it is clearer to put something like "Author Site" as the old code.